.github/ISSUE_TEMPLATE/source-distribution.yml

```diff
@@ -0,0 +1,34 @@
+---
+name: "Source distribution issue"
+description: Report an issue with source distribution
+title: "(short issue description)"
+labels: [source-distribution, needs-triage]
+assignees: []
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the issue
+      description: What is the problem? A clear and concise description of the issue.
+    validations:
+      required: true
+  - type: textarea
+    id: context
+    attributes:
+      label: Additional Information/Context
+      description: |
+        Anything else that might be relevant for troubleshooting this issue.
```

Comment:
It might be good to suggest context that would be useful here such as `configure` options used. I'd imagine we'd almost always want that.